packages/core/docs/core/api-client.md

```diff
@@ -4,7 +4,7 @@
 
 ----
 ::: tip Installation
-If you want to learn how to install the API Client check our [Getting Started guide](/getting-started.html)
+If you want to learn how to install the API Client check our [Installation guide](/general/installation.html)
```

Comment:
I think this is no longer relevant (and the doc is not even used, can be deleted)